package.json

```diff
@@ -107,7 +119,10 @@
         }
     },
     "dependencies": {
```

Comment:
You should not have `npm` dependencies in this project. All dependencies using from npm should be `devDependencies`. All dependencies that you use in the production, should be loaded via JSPM.